### ISSUE TITLE
No Z-Axis available for Laser

### ISSUE THREAD

**Issue by 2nice4u** [makerbase-mks/MKS-DLC32#333]
I updated the Laser V2.1 to the latest V2.3 and now there is no option to move the z axis!
Now I have no way to move the height-adjustable table.

![No-Z](https://user-images.githubusercontent.com/117979557/215446134-31df2071-18e4-4e46-9800-480ddb44bc96.jpg)

**Comment by philvtx** [comment]
Bonjour,
Oui j’ai le même problème mais surtout je n’ai qu’ 1 axe qui prends les références x ou y le plus près du switch !
Le Z aucune référence ne se fait ! J’ai flashé toutes les versions et toujours le même problème ! 
Carte MKS DLC32 ET LIGHTBURN

**Comment by abinoer** [comment]
Yes, I'm sorry, this version does not plan Z-axis control

**Comment by 2nice4u** [comment]
> Yes, I'm sorry, this version does not plan Z-axis control

But I'm not the only one with a height-adjustable table via a stepper motor. The versions before that had offered this possibility. I see this as a step backwards.

**Comment by cyberjak2k** [comment]
I agree this was a downgrade , we need the option of Z axis control otherwise board is not useful

**Comment by cyberjak2k** [comment]
> Yes, I'm sorry, this version does not plan Z-axis control

can this please be revisited, this version fixed issue with compatibility with mobile app but removed Z axis control REALLY need this back or it means board no longer fit for purpose and might require return which I'd rather not do as I liked the idea of it

**Comment by Robhoski** [comment]
Agree! I use the MKS DLC32 V2.1 to run my plasma cutter. I NEED Z_AXIS! The workaround sucks - gotta sero the z-axis in software first. PIA

**Comment by cyberjak2k** [comment]
Yes it was removed in a previous version I agree this should be restored

**Comment by thomonz** [comment]
Miss the z axis gor the laser version. Can this be re instated

**Comment by cyberjak2k** [comment]
I think only by reverting to older version which I agree is a downfall
